src/Discord.Net.Core/Entities/Channels/IAudioChannel.cs

```diff
@@ -8,5 +8,11 @@ public interface IAudioChannel : IChannel
     {
         /// <summary> Connects to this audio channel. </summary>
         Task<IAudioClient> ConnectAsync(Action<IAudioClient> configAction = null);
+
+        /// <summary> Connects to this audio channel but can specify if client is handled externally. </summary>
+        Task<IAudioClient> ConnectAsync(Action<IAudioClient> configAction = null, bool external = false);
```

Comment:
Wouldn't this cause an error when you call `ConnectAsync((client) => {});`? I'd suggest you merge both of these methods.